libpod/top.go

```diff
@@ -0,0 +1,37 @@
+package libpod
```

Comment:
Nit: can you rename this file to container_top.go

Reply:
done